pkg/pb/synthetic_monitoring/checks_extra_test.go

```diff
@@ -728,6 +728,55 @@ func TestValidateLabel(t *testing.T) {
 	}
 }
 
+func TestHttpSettingsValidate(t *testing.T) {
+	testcases := map[string]struct {
+		input       HttpSettings
+		expectError bool
+	}{
+		"trivial": {
+			input:       HttpSettings{},
+			expectError: false,
+		},
+		"valid headers": {
+			input: HttpSettings{
+				Headers: []string{"header: value"},
+			},
+			expectError: false,
+		},
+		"no value is OK": {
+			input: HttpSettings{
+				Headers: []string{"header:"},
+			},
+			expectError: false,
+		},
+		"empty": {
+			input: HttpSettings{
+				Headers: []string{""},
+			},
+			expectError: true,
+		},
+		"no colon": {
```

Comment:
let's add a tests for:
- only colon with space around it (`" : "`)
- unicode (emojis or non english text) in header values (`header: यूनिकोड टेक्स्ट`)

> यूनिकोड टेक्स्ट says `Unicode text` in Hindi language